### PR TITLE
Change column names for per sample and per project metrics

### LINKED ISSUE #59: Improve / change column names for per sample and per project metrics
For `per_sample_metrics.tsv`
- [ ] Change `barcode_name` to `sample_ID`
- [ ] Change `templates` to `total_matches`

For `per_project_matches`
- [ ] Change `templates` to `total_matches`

### PR DESCRIPTION
**Pull request by theJasonFan**
Completes tasks in #59

Todos:
- [ ] add to a changelog or release notes. Changes to struct field members and serialized TSV cols constitute a breaking change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -359,7 +358,7 @@ The `per_sample_metrics.tsv` file produces a row per sample.
 
 The `per_project_metrics.tsv` file aggregates the metrics by project (aggregates the metrics across
 samples with the same project) and has the same columns as [per_sample_metrics.tsv](#per_sample_metricstsv).
-In this case, `barcode_name` and `library_name` will contain the project name (or `None` if no
+In this case, `sample_ID` and `library_name` will contain the project name (or `None` if no
```
Comment (nh13) [inline README.md:361]: Can you search for `library_name` here and through-out the code to ensure we've removed it?

**Review by nh13 — CHANGES_REQUESTED** [review]
One minor thing to do.

File: README.md (hunk 1 above)
Comment (theJasonFan) [reply README.md]: Fixed now.

**Review by nh13 — APPROVED** [review]

**Review by jiangweiyao — APPROVED** [review]

**Comment by theJasonFan** [comment]
Rebased onto main, merging in changes from #60.

**Pull request merged**